compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -115,7 +113,14 @@ interface PlatformContext {
     val parentFocusManager: FocusManager get() = EmptyFocusManager
     fun requestFocus(): Boolean = true
 
-    fun createDragAndDropManager(): PlatformDragAndDropManager = EmptyDragAndDropManager
+    /**
+     * Starts a drag-and-drop session with the Compose app as the source of the transfer.
+     */
+    fun startDrag(
+        transferData: DragAndDropTransferData,
+        decorationSize: Size,
+        drawDragDecoration: DrawScope.() -> Unit
+    ): Boolean = false
```

Comment:
This is not compatible with our common design and other platforms. Subject to change.